cypress/integration/404.spec.ts

```diff
@@ -0,0 +1,23 @@
+it('should display custom 404 page when not found', () => {
+  cy.visit('/not-existing-page', { failOnStatusCode: false })
```

Comment:
I've set the `failOnStatusCode` here so I can test the custom 404 only in this very test.

Reply:
Cool!